src/test/java/org/influxdb/InfluxDBTest.java

```diff
@@ -891,5 +895,63 @@ public void testMessagePackOnOldDbVersion() {
 	    influxDB.describeDatabases();
 	  });
 	}
-	
+
+  /**
+   * test for issue #445
+   * make sure reusing of OkHttpClient.Builder causes no error
+   * @throws InterruptedException
+   */
+  @Test
+  public void testIssue445() throws InterruptedException {
+    ExecutorService executor = Executors.newFixedThreadPool(100);
+
+    final int maxCallables = 10_000;
+    List<Callable<String>> callableList = new ArrayList<>(maxCallables);
+    for (int i = 0; i < maxCallables; i++) {
+      callableList.add(new Callable<String>() {
+        @Override
+        public String call() throws Exception {
+          MyInfluxDBBean myBean = new MyInfluxDBBean();
+          return myBean.connectAndDoNothing1();
+        }
+      });
+    }
+    System.out.println("Invoking all callableList (size()=" + callableList.size() + ")");
```

Comment:
I am not a big fan of having sysout in tests, are these necessary ?

Reply:
removed them